src/components/Header.js

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link'
+import ProfileIcon from './ProfileIcon'
+
+class Header extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      name: 'evilfactorylabs blog',
+      avatar: 'https://avatars1.githubusercontent.com/u/43804217?s=200&v=4',
+    }
+  }
+
+  render() {
```

Comment:
We can use Functional component style like this for consistency (test needed)

```suggestion
const Header = () => {
  // since I don't see any code that *mutating* the state, I assume this is not a dynamic data
  const state = {
    name: 'evilfactorylabs blog',
    avatar: 'https://avatars1.githubusercontent.com/u/43804217?s=200&v=4',
  }
  
```

Reply:
Haha bener gan, yang ini emang ga akan diubah2 state nya.
Ane mikirnya ini data2 adalah status login yg didapet dari backend. Skrg ya dummy state aja.
Oke ane ganti ke func comp

Reply:
bisa langsung apply changes nya disini gan, via commit suggestion